Mathlib/GroupTheory/GroupAction/SubMulAction.lean

```diff
@@ -133,8 +133,8 @@ variable {N α : Type*} [SetLike S α] [SMul M N] [SMul M α] [Monoid N]
 -- lower priority so other instances are found first
 /-- A subset closed under the scalar action inherits that action. -/
 @[to_additive "A subset closed under the additive action inherits that action."]
-instance (priority := 50) smul' : SMul M s where
-  smul r x := ⟨r • x.1, smul_one_smul N r x.1 ▸ smul_mem _ x.2⟩
+instance (priority := 50) smul' : SMul M s :=
+  smul r x := ⟨r • x.1, have := SMulMemClass.ofIsScalarTower S M N α; smul_mem _ x.2⟩
```

Comment:
```suggestion
instance (priority := 50) smul' : SMul M s where
  smul r x := ⟨r • x.1, smul_one_smul N r x.1 ▸ smul_mem _ x.2⟩
```
since this is out of scope for this PR anyway.